client/app/lib/api-utility.js

```diff
@@ -50,13 +51,17 @@ export const post = async (uri, requestBody) => {
 
         // Builds the full url to be called during fetch()
         const url = API_BASE_URL + uri;
-
         // Run our HTTP POST using fetch
         const response = await fetch(url, postConfig);
-
+        
         // If the response returned a status code that's not 200 (OK)
         // Throw an error with the status code text.
-        if (!response.ok) throw new Error(response.statusText);
+
+        if (!response.ok){
+            
```

Comment:
why all the space here?

Reply:
Left over from debugging I think, I'll go in and fix it